specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/sqlServer.json

```diff
@@ -1188,6 +1188,185 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Synapse/workspaces/{workspaceName}/dedicatedSQLminimalTlsSettings/{dedicatedSQLminimalTlsSettingsName}": {
+      "put": {
+        "operationId": "WorkspaceManagedSqlServerDedicatedSQLminimalTlsSettings_Update",
+        "summary": "Update server's tls settings.",
+        "description": "Update workspace managed sql server's minimal tls settings.",
+        "tags": [
+          "WorkspaceManagedSqlServer, DedicatedSQLminimalTlsSettings"
+        ],
+        "deprecated": false,
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../common/v1/types.json#/parameters/WorkspaceNameParameter"
+          },
+          {
+            "name": "dedicatedSQLminimalTlsSettingsName",
+            "in": "path",
+            "description": "The name of the dedicated sql minimal tls settings.",
+            "required": true,
+            "type": "string",
+            "enum": [
+              "default"
+            ],
+            "x-ms-enum": {
+              "name": "DedicatedSQLMinimalTlsSettingsName",
+              "modelAsString": true
+            }
+          },
+          {
+            "schema": {
+              "$ref": "#/definitions/DedicatedSQLminimalTlsSettings"
+            },
+            "in": "body",
+            "name": "parameters",
+            "description": "minimal tls settings",
+            "required": true
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully updated dedicated sql minimal tls settings.",
+            "schema": {
+              "$ref": "#/definitions/DedicatedSQLminimalTlsSettings"
+            }
+          },
+          "202": {
+            "description": "Accepted"
+          },
+          "default": {
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            },
+            "description": "*** Error Responses: ***\n\n * 404 SubscriptionDoesNotHaveServer - The requested server was not found.\n\n * 404 ServerNotInSubscriptionResourceGroup - Specified server does not exist in the specified resource group and subscription.\n\n * 409 UnsupportedWorkspaceType - Unsupported operation for the workspace type.\n\n * 500 InternalServerError - Something went wrong."
```

Comment:
>SubscriptionDoesNotHaveServer [](http://example.com/codeflow?start=63&length=29)

what server is this referring to?

Reply:
@rkmanda : This is related to workspace sql server.

Reply:
@rkmanda : Can you please check and resolve the comment